workshops/views.py

```diff
@@ -521,6 +520,67 @@ def person_password(request, person_id):
     })
 
 
+@login_required
+def person_merge(request):
+    '''Merge information from one Person into another (in case of duplicates).'''
+
+    if request.method == 'POST':
+        form = PersonMergeForm(request.POST)
+        if form.is_valid():
+            # FIXME: what should be serialized?
+            request.session['person_from'] = form.fields['person_from']
+            request.session['person_to'] = form.fields['person_to']
```

Comment:
I have no strong preference for either. I try to use `.pk` because it's more general, but I'm pretty sure we won't change the default primary key from `id` to anything else.